docs/knowledgebase/getting-started/index.md

```diff
@@ -74,119 +74,69 @@ rustup default stable
 
 Substrate uses [WebAssembly](https://webassembly.org/) (Wasm) to produce portable blockchain
 runtimes. You will need to configure your Rust compiler to use
-[`nightly` builds](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) to allow
-compiled substrate compatible runtimes to Wasm.
+[`nightly` builds](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) to allow you to
+compile Rust code to the Wasm target.
 
 #### Rust Nightly Toolchain
 
-Because the nightly toolchain is a moving target and receives daily changes the chance
-that some of them break the substrate build from time to time is non-negligible.
-
-Therefore it is advised to use a fixed nightly version rather than the latest one to
-build the runtime. You can install a specific version using this command:
+Developers that are building _with_ Substrate (as opposed to the developers building Substrate
+_itself_) should use a specific Rust nightly version that is known to be compatible with the version
```

Comment:
This distinction of _with_ and _itself_ is confusing to me. There is no technical difference. For that reason I specifically removed it.

Reply:
Substrate master moves with the nightly versions, this means if a future version breaks something we fix it on master. This often makes Substrate master fail to compile with an older nightly. So, if you are using Substrate 2.0.0 they should use a specific nightly version. However, if you develop on Substrate master you will probably need to update your nightly regularly.

Reply:
So then we should write that instead. Working with master vs. working with a release. You need to tell people the **why** or they will ignore your advice.

Reply:
You can write whatever you want, people will still ignore it xD :D

Reply:
yes I agree with @athei the distinction is confusing and just creates more questions than answers.